sandbox-templates/expo-android/start_android.sh

```
@@ -0,0 +1,47 @@
+adb wait-for-device
+
+# Wait for boot completion
+echo "[INFO] Waiting for boot completion..."
+while [[ -z $(adb shell getprop sys.boot_completed 2>/dev/null | tr -d '\r') ]]; do
+    sleep 2
+done
+
+echo "[INFO] Emulator ready!"
+
+# Start Expo Metro bundler with Android
+cd /home/user
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)